src/pages/tasks/NewTaskPage.tsx

```diff
@@ -97,8 +94,8 @@ function NewTaskPage({route}: NewTaskPageProps) {
         }
 
         playSound(SOUNDS.DONE);
-        TaskActions.createTaskAndNavigate(
-            parentReport?.reportID ?? '-1',
+        createTaskAndNavigate(
+            parentReport?.reportID ?? CONST.REPORT.DEFAULT_REPORT_ID,
```

Comment:
I am not sure if adding the string `DEFAULT_REPORT_ID` is the right approach? What happens if we allow string|undefined?

Reply:
we recently updated guidance on default values so checking on this